src/radio.cpp

```diff
@@ -100,18 +100,18 @@ void zmq::radio_t::xwrite_activated (pipe_t *pipe_)
 
 void zmq::radio_t::xpipe_terminated (pipe_t *pipe_)
 {
-    for (subscriptions_t::iterator it = subscriptions.begin (); it != subscriptions.end (); ) {
```

Comment:
Standard implementation unfortunately have bugs or subtle differences on decades-old platforms and that caused a regression. Unfortunately it was almost a year ago so I can't remember exactly anymore, but I think it was on Solaris 10